.github/workflows/macos11.yml

```diff
@@ -0,0 +1,46 @@
+run-name: macOS-11_${{ vars.current_date }}_unstable.${{ github.run_id }}.${{ github.run_attempt }}
+on:
+  workflow_dispatch:
+    inputs:
+      CUSTOM_REPOSITORY:
+        description: 'Custom repository (owner/repo)'
+        required: false
+      CUSTOM_REPOSITORY_HASH:
+        description: 'Commit hash'
+        required: false
+  pull_request_target:
+    types: labeled
+    paths:
+    - 'images/macos/**'
+  schedule:
+    - cron: '0 0 * * *'
+
+jobs:
+  update_vars:
+    if: ${{ (github.event.label.name == 'macos-all') || (github.event.label.name == 'macos-11') || (inputs.CUSTOM_REPOSITORY != '' && inputs.CUSTOM_REPOSITORY_HASH != '') || (github.event_name == 'schedule') }}
+    name: Update Variables
+    runs-on: windows-latest
+    steps:
+      - name: run api
+        shell: pwsh
+        run: |
+          $currentDate = Get-Date -Format "yyyyMMdd"
+          $uri = "https://api.github.com/repos/AleAiu/simpletest/actions/variables/CURRENT_DATE"
+          $header = @{"Authorization" = "Bearer ${{ secrets.REPO_TOKEN }}"}
+          $body = "{`"name`":`"CURRENT_DATE`",`"value`":`"$currentDate`"}"
+          Invoke-WebRequest -Method PATCH -Header $header -Uri $uri -Body $body -ContentType "application/json"
+
+  macOS_11:
+    if: ${{ (github.event.label.name == 'macos-all') || (github.event.label.name == 'macos-11') || (inputs.CUSTOM_REPOSITORY != '' && inputs.CUSTOM_REPOSITORY_HASH != '') || (github.event_name == 'schedule') }}
```

Comment:
Do we really need to repeat the full condition here?
This job depends on the first time so it should be auto-cancelled if first one is skipped.